### DISCUSSION TITLE
Custom text for Row Selection

### DISCUSSION THREAD

**Discussion by CamachoPt** [KevinVandy/material-react-table#294] [Q&A] [answered]
Is it possible to create a custom text for row selection ?

For example having "2 Results ( 1 selected )" insead of the normal "1 of 2 row(s) selected" ?

**Comment by CamachoPt** [comment]
Edit : Got it with "renderTopToolbar"

**Comment by KevinVandy** [comment, marked as answer]
It depends how much you want to override. If you use `renderTopToolbar`, you loose all of the built-in functionality of all the top toolbar.

You can just use `muiToolbarAlertBannerProps.children` to change the exact message in the toolbar.

This can be seen in the [Remote Example](https://www.material-react-table.com/docs/examples/remote) on the docs site to show a custom error message.

Another option would be to change the `localization.selectedCountOfRowCountRowsSelected` prop to be that format.
